nginx/nginx.tmpl

```diff
@@ -102,22 +99,28 @@ http {
     }
 {{ end }}
 
+{{- $IngressPorts := .Ports }}
+{{- $SSLPath := .SSLPath }}
+{{define "HTTPSConf"}}
+        # https://mozilla.github.io/server-side-tls/ssl-config-generator/ - Nginx, Modern Profile + TLSv1, TLSv1.1
+        ssl_certificate {{ . }}.crt;
+        ssl_certificate_key {{ . }}.key;
+        ssl_session_timeout 1d;
+        ssl_session_cache shared:SSL:50m;
+        ssl_session_tickets off;
+        ssl_protocols TLSv1 TLSv1.1 TLSv1.2;
```

Comment:
Can we use just TLSv1.2 until we run into problems? All modern clients that we're accepting should use this. (Not sure on this, welcome to opposing viewpoints). I'm mostly concerned about straying from the "modern profile" with our own customizations (might be some security issues with this we're not aware of).

Reply:
Depending if some client use IE lower than 11, well they should be deprecated. maybe we can only use TLSv1.2.

Reply:
I don't think old IE will work with this configuration anyways - not sure it supports the ciphers we want to use.